app/controllers/api/messaging.php

```diff
@@ -1230,7 +1217,7 @@
     });
 
 App::delete('/v1/messaging/providers/:providerId')
-    ->desc('Delete Provider')
+    ->desc('Delete provider')
     ->groups(['api', 'messaging'])
     ->label('audits.event', 'providers.delete')
```

Comment:
Let's update this label to singular form too

Reply:
done this